coordinator/src/app_desc/bytes.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
```

Comment:
The only use of this `Bytes` thing in the context of app desc is just to deserialize `[u8]` from a long sequence of hexadecimal digits. I wrote a helper function to achieve the same in parsing app desc, and now I feel we require such a functionality here and there and are duplicating similar code. In this sense, we need to put it where appropriate (maybe in a separate crate) and reuse where relevant.

Also I'm not sure a new type here really adds any value here other than affecting serialization/deserialization.

I might need to search a bit more to find a crate that fits best. But https://docs.rs/serde_with/1.5.1/serde_with/hex/struct.Hex.html seems a good candidate.

Reply:
Thanks. I just moved it from the scheme code. I'll add a commit that replace it using the serde_with crate.

Reply:
Actually I also have such a parsing facility at `coordinator/src/app_desc.rs:235`. It should be replaced altogether. Once you are done, I'll do the similar.

Reply:
@majecty @junha1 uses `hex` crate and it also serves the hex conversion quite well. Let's use it rather than `serde_with` if our main concern is to deal with hex representation of byte sequence.